packages/core/src/entities/InlineTool.ts

```diff
@@ -0,0 +1,60 @@
+import type { InlineTool as InlineToolVersion2 } from '@editorjs/editorjs';
+import type { BlockToolConstructorOptions as BlockToolConstructorOptionsVersion2 } from '@editorjs/editorjs';
+import type { TextRange, InlineFragment, FormattingAction, IntersectType, InlineToolName } from '@editorjs/model';
+
+/**
+ * Extended BlockToolConstructorOptions interface for version 3.
+ */
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+export interface InlineToolConstructorOptions extends BlockToolConstructorOptionsVersion2 {}
+
+/**
+ * Block Tool interface for version 3
+ *
+ * In version 3, the save method is removed since all data is stored in the model
+ */
+export interface InlineTool extends Omit<InlineToolVersion2, 'save' | 'checkState' | 'render'> {
+  /**
+   * Name of the inline tool
+   */
+  name: InlineToolName;
+
+  /**
+   * Type of merging of two ranges which intersect
+   */
+  intersectType: IntersectType;
```

Comment:
it should be optional and have a default value

Reply:
is this related to config also?